Confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -74,6 +81,7 @@ class Confidence internal constructor(
     }
 
     fun apply(flagName: String, resolveToken: String) {
+        debugLogger?.logFlags("ApplyFlag", flagName)
```

Comment:
here we are calling the log function before the apply and in the context functions we call the logging after the operation. do we want to be consistent with the meaning of the logs we provide? do they mean we have called and done apply or resolve or it means we are about to start the operation?

Reply:
oh great point. i think we should call the logs after we do the operation, that would be consistent with what i had in mind. thanks for pointing it out!